bundle/tests/scorecard/kuttl/persistent-storage/00-persistent-storage-rc.yaml

```diff
@@ -8,12 +8,13 @@ spec:
   volumeMounts:
```

Comment:
we should add a test to make sure StatefulSet is created and not Deployment when `statefulSet: {}` is specified in CR. Fine with doing that via a separate PR.